internal/security/bootstrapper/command/waitfor/uri_flags.go

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package waitfor
+
+import "fmt"
+
+type uriFlagsVar []string
+
+func (uris *uriFlagsVar) String() string {
+	return fmt.Sprint(*uris)
+}
+
+func (uris *uriFlagsVar) Set(value string) error {
```

Comment:
Should this be named `Add` rather than `Set`? To me `Set` would overwrite and this is adding.

Reply:
No, this Set is the method to set the flag value, part of the `flag.Value` interface from Golang Flag.

Reply:
See https://golang.org/pkg/flag/#Value

Reply:
Added comments implementing Golang's flag.Value interface.

Reply:
K